scripts/download_artifacts.py

```diff
@@ -0,0 +1,56 @@
+import argparse
```

Comment:
Using the github rest api to download the artifacts created in the "build pr artifacts" workflow.

There is [technically](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-data-from-the-triggering-workflow) a way to do it directly on the github actions but that requires using js and it's harder to test/debug in my opinion. I couldn't get it to work.

Reply:
👍